File: ACTUALLYGAME.py
from random import randint 
z = randint(1,100)
print("This is a game named Number Guesser ")
print("In this Game you have to choose a Number between 1-100")
print("So,lets begin ")
print("There are three levels")
print('''==========WELCOME==========
1 Noobs level 
2 Pro level
3 God level''')
x = int(input("Choose your level\n"))
if (x==1):
    print('In this level you will get to know that wether you choose greater number or smaller number ')
    a = int(input("Write your Number\n"))
    if(a==z):
        print("Why you played this level you are actually a pro")
    while a!=z:
        if (a>z):
            print("You choosed a Greater number try  to choose a smaller number")
        else:
            print("You choosed a Smaller number try  to choose a Greater number")   
        a = int(input("Write your Guess again\n"))   
        if(a==z):
            print("Finally you choosed a write number Good :)")
elif (x==2):
    print("In this level  you will get unlimited turns but you will not get to know wether the number you choosed is greter or smaller than the actual number")              
    a = int(input("Write your Guess \n")) 
    if (a==z):
        print("You are actually a Pro!\nYOU WON!")
    while (a!=z):
        a = int(input("Write your Guess Again \n")) 
    if (a==z):
        print("Finally You Choosed a write number")    
elif(x==3):
    print('Well you think you are a God \nLets see')
    print ("You have only one chance so be carefull while choosing a number :) All The Best!")
    a = int(input("Write your Guess \n"))   
    if (a==z):
        print ("You Are Actually A God in this game!")
        print("congratualtions\n"*10)
    else:
        print("You are a NOOB!\n"*20)
else :
    print("Choose a valid Level")
if (x==1 or x==2 or x==3):
    print("THANKS FOR PLAYING :)")    
            
  

File: ACTUALLYGAME.java
import java.util.*;
class gamessssss
{
    public static void main (String ar[]){
    Scanner sc = new Scanner (System.in);
    Random rn = new Random();
    int r = rn.nextInt(100); 
     System.out.println("This is a Simple number guessing game\nYou have to guess a number between 0-100\n______________________________________________________________________\n==========Let's Begin==========\nChoose your level\n1.Noobs level\n2.Pro level\n3.Gods Level\n");
    int level = sc.nextInt();
    if (level==1){
        System.out.println("In this level you will get to know that wether the number you choose is greater or smaller than the actual number");
        System.out.println("Write Your Number \n");
        int c = sc.nextInt();
        while (r!=c){
            if (r>c){
                System.out.println("You choosed a smaller number \nTry to choose a greater number \nChoose again");
                int d = sc.nextInt();
                c=d; 
            }
            else if (r==c){//Ignore this sentence (else if )if you are reading by mistake it was added
                System.out.println("You Won");
            }
            else{
                System.out.println("You choosed a greater number \nTry to choose a smaller number\nChoose again");
                int e = sc.nextInt();
                c=e;
            }
        }
        if (r==c){
            
            System.out.println("You Won!");
        }
    
    
    
    }
    if (level==2){
        System.out.println("So You are a Pro \nYou will get unlimited turns but will not be told that wether the nubmer you choose is greater or smaller");
        System.out.println("Write your Numer\n");
        int c = sc.nextInt();
        while (c!=r){
            
            System.out.println("Choose A number again\nWrite here\n");
            int z = sc.nextInt();
            c = z;
         }
    if (c==r){
        System.out.println("You Won!");
    }   

    }
    if (level==3){
        System.out.println("So you think you are a god \nYou will get only one choice \nWrite the number you choose\n");
        int c = sc.nextInt();
        if(c==r){
            System.out.println("You are Actually a God !\nYOU WON!");
        }
        else{
            System.out.println("You are not a god Looser!");
        }
    }
}
}
